src/main/java/org/cactoos/scalar/ScalarWithFallback.java

```diff
@@ -78,11 +78,11 @@ public ScalarWithFallback(
     public ScalarWithFallback(
         final Scalar<T> origin,
         final Iterable<Class<? extends Throwable>> exceptions,
-        final Func<Throwable, T> fallback
+        final Fallback<T> fallback
```

Comment:
@baudoliver7 let's just remove all the constructors taking exception+fallback and only keep those taking Fallback. If people want they can just use `FallbackFrom` directly themselves.